module/zfs/arc.c

```diff
@@ -7737,12 +7871,19 @@ arc_init(void)
 
 	buf_init();
 
+	if (zfs_arc_evict_threads == 0)
+		zfs_arc_evict_threads = MIN(16, max_ncpus >> 1);
```

Comment:
It is subjective, but IMHO 1/2 of CPUs for memory eviction is too much if system is doing anything else.  And the current algorithm for non-trivial eviction would tend to use all of them quite often, stressing the system.